packages/theme-chalk/src/input-number.scss

```diff
@@ -27,6 +27,16 @@
       }
       text-align: center;
     }
+
+    &__prefix {
+      position: absolute;
+      left: #{map.get($input-height, 'default') + 10};
+    }
+
+    &__suffix {
+      position: absolute;
+      right: #{map.get($input-height, 'default') + 10};
+    }
```

Comment:
It seems enough to remove this line,
https://github.com/element-plus/element-plus/blob/33ae857c68e0ef625cfe177b75855def3e9d0810/packages/theme-chalk/src/input-number.scss#L15

Reply:
I'm not sure if this line does something else.

Because after removing it, it will become inline-flex

Reply:
yes, thanks